meson.build

```diff
@@ -467,6 +467,16 @@ foreach it : ccs
     it_userconf.set10('HAVE_HEADER_@0@'.format(hdr.underscorify().to_upper()), ok)
   endforeach
 
+  foreach intrin : [
```

Comment:
There is already a foreach loop that does this so you can add these functions there

Reply:
Should I rename the macro to have the same name as the function (followed by all other macros) or do I keep them the same as before ?.  ( i.e. `HAVE_BSWAP16` -> `HAVE___BUILTIN_BSWAP16` ?)

Reply:
Yeah, `HAVE___BUILTIN_BSWAP16` is better